tests/PHPStan/Rules/Comparison/MatchExpressionRuleTest.php

```diff
@@ -153,7 +153,11 @@ public function testEnums(): void
 				56,
 			],
 			[
-				'Match arm comparison between *NEVER* and MatchEnums\Foo is always false.',
+				'Match arm comparison between MatchEnums\Foo::THREE and MatchEnums\Foo::THREE is always true.',
+				76,
+			],
+			[
+				'Match arm is unreachable because previous comparison is always true.',
```

Comment:
in contrast, I like this error way more then the one we had before this PR

Reply:
👍